src/EditModel/event-program/epics.js

```diff
@@ -449,7 +448,7 @@ export const programModelSave = action$ =>
         .switchMap(({ eventProgramStore, missingFields }) => (
             missingFields.length
                 ? createSaveEventProgramError$()
-            
+
```

Comment:
:1st_place_medal:

Reply:
Yeah I think that that's atom stripping out trailing whitespace. It's a bit annoying. That's why I wanted to add an editorconfig to code-style, so that I can point to that as the culprit 😋

I could remove this btw. It's nice to not have trailing whitespace, but strictly speaking probably nicer to do that separately?

Reply:
I think it's fine to remove trailing whitespace in a commit. Larger refactoring work might warrant a separate commit though.

Reply:
I'd argue that this a good thing actually. Almost all editors strip trailing white spaces.
When hitting 'end' or `$` in vim, I want to go to the last visible character, not to somewhere unexpected.

Reply:
Oh yeah, don't get me wrong, I like having an editorconfig on all my projects to remove trailing whitespace, adding end of file newlines, etc. It's just that I thought some might consider it messy if it's not the subject of the pr. But I'm happy that noone minds it :)